hrw.go

```diff
@@ -83,6 +83,17 @@ func SortSliceByValue(slice interface{}, hash uint64) {
 	}
 }
 
+// SortHasherSliceByValue received []Hasher and hash to sort by value-distance
```

Comment:
s/received/receives/
dot in the end of the comment: there is a linter which complains + Godoc is nicer.

Reply:
fixed